infra/license/README.md

```diff
@@ -0,0 +1,71 @@
+# License Verification
+
+## Background
+
+Numerous kinds of Node packages are needed to build and use ONE-vscode but some packages may have unacceptable license for both ONE and ONE-vscode.
+Therefore we should review whether licenses of each package can be used or not.
+However, if we check all of them manually, we may mistake or miss something.
+To reduce such risk and labor, automatic license checker would be good solution.
+
+## Process & Architecture
+
+### Flowchart
+
+![flowchart](flowchart.png)
+
+### Terminology
+
+- license-judgment.json
+  - Each of license names are got from the result of `license-checker`
+  - Each license is classfied to following categories
+    - `Allowed` : Allowed to use
+    - `Warning` : Manual review for each package is needed
+    - `Denied` : Not allowed to use
+- package-judgment.json
+  - Each judgment for a package is consisted of following information
+    ```json
+    "pkgname@version": {
+      "licenses": "License name",
+      "repository": "URL for package repository",
+      "ONE_permitted": "yes / conditional / no",
+      "ONE_caution": "(Optional field) Cautions when package is used"
+    },
+    ```
+    - `ONE_permitted` : Whether the package is allowed in ONE-vscode
```

Comment:
(optional) `ONE` prefix is needed? Just `is_permitted` and `caution` looks okay to me.

Reply:
I think your suggestion seems reasonable :)
Since #292 is already using `ONE_*`, I will create another PR to remove the prefix at once!